NIPTool/build/document.py

```diff
@@ -0,0 +1,34 @@
+from NIPTool.models.constants import SAMPLE_KEYS, BATCH_KEYS
+from NIPTool.models.converters import convert
+
+
+def build_document(csv_data: dict, document_keys: list) -> dict:
+    """Build a general document based on convert models"""
+
+    document = {}
+    for key in document_keys:
+        value = csv_data.get(key)
+        value = convert(key, value)
+        if value is None:
+            continue
+        document[key] = value
+
+    return document
+
+
+def build_sample(sample_data: dict) -> dict:
+    """Builds a document for the sample collection"""
+
+    sample = build_document(sample_data, SAMPLE_KEYS)
+    sample["_id"] = sample_data.get("SampleID")
+
+    return sample
+
+
+def build_batch(batch_data: dict) -> dict:
+    """Builds a document for the batch collection"""
+
+    batch = build_document(batch_data, BATCH_KEYS)
+    batch["_id"] = str(batch_data.get("SampleProject"))
```

Comment:
If "SampleProject" does not exist in batch_data this will be set to the string `"None"`, I guess that is not what one wants..

Reply:
It is being handeld allready in parse. Same as above!

Reply:
```suggestion
    batch["_id"] = str(batch_data["SampleProject"])
```
Same here!

Reply:
Good point! 👍